homeassistant/components/notify/pushover.py

```diff
@@ -63,13 +63,21 @@ def send_message(self, message='', **kwargs):
 
         targets = kwargs.get(ATTR_TARGET)
```

Comment:
This would indeed prevent the crash but would cause the loop to never run and no notifications to be sent. We need to instead send the generic notification call.